src/js/components/Books/Book.scss

```diff
@@ -1,8 +1,16 @@
-.books{
+.Books{
     background-color: #555;
 }
 
-.no-padding-margin{
+.bookCardFrame{
+    min-height: 150px;
+}
+
+.minHeight150{
+    min-height: 150px;
+}
+    
+.noPaddMar{
     padding: 0px ;
-    margin: 0px;
+    margin: 0;
```

Comment:
keep consistent if you are specifying 0px or 0
I suggest with zero values do not specify px
Add an empty line at the end of file and the next scss files

Reply:
thank you I'ts updated.